src/main/io/gps.c

```diff
@@ -1252,9 +1346,9 @@ static bool _new_speed;
 //15:17:55  R -> UBX NAV,  Size 100,  'Navigation'
 //15:17:55  R -> UBX NAV-SVINFO,  Size 328,  'Satellite Status and Information'
 
-// from the UBlox6 document, the largest payout we receive i the NAV-SVINFO and the payload size
-// is calculated as 8 + 12*numCh.  numCh in the case of a Glonass receiver is 28.
-#define UBLOX_PAYLOAD_SIZE 344
+// from the UBlox9 document, the largest payout we receive is the NAV-SAT and the payload size
+// is calculated as 8 + 12*numCh.  numCh in the case of a M9N is 42.
+#define UBLOX_PAYLOAD_SIZE (8 + 12*42)
```

Comment:
Please give operators some room, also in other places.
```suggestion
#define UBLOX_PAYLOAD_SIZE (8 + 12 * 42)
```

Reply:
done